app/uk/gov/hmrc/customs/notification/domain/CustomsNotificationConfig.scala

```diff
@@ -25,6 +25,7 @@ case class NotificationQueueConfig(url: String)
 
 case class PushNotificationConfig(
```

Comment:
I think we need a similar switch for the legacy module `FailedPushEmailPollingService`